pvlib/iotools/bsrn.py

```diff
@@ -217,7 +220,7 @@ def get_bsrn(station, start, end, username, password,
     return data, metadata
 
 
-def parse_bsrn(fbuf, logical_records=('0100',)):
+def _parse_bsrn(fbuf, logical_records=('0100',)):

```

Comment:
Is there a reason that we aren't putting this content in the read function and completely get rid of the parse function? It is a bit of a hassle, but long term probably worth it it seems.

Reply:
After posting that comment, I realized that might be the case. Fine by me.